dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -306,15 +306,15 @@ BEGIN
     LTEXT "&Parola:", 1112, 10, 81, 92, 8
     EDITTEXT 1103, 104, 77, 154, 14, ES_PASSWORD | ES_AUTOHSCROLL
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN, 10, 97, 247, 1
-    AUTOCHECKBOX "Păstrea&ză acest nume și parolă pentru acești utilizatori:", 1101, 12, 103, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
+    AUTOCHECKBOX "Sal&vare a acestor nume și parolă pentru acești utilizatori:", 1101, 12, 103, 243, 13, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru mine, e&xclusiv", 1622, 24, 117, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE | WS_GROUP
     AUTORADIOBUTTON "Pentru &oricine din acest calculator", 1623, 24, 132, 229, 11, BS_LEFT | BS_TOP | BS_MULTILINE
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 147, 247, 1
     LTEXT "Ap&elează:", 1412, 13, 157, 88, 8
     COMBOBOX 1416, 103, 154, 154, 112, CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | NOT WS_TABSTOP
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 173, 247, 1
-    DEFPUSHBUTTON "&Apelează", 1590, 10, 180, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 72, 180, 60, 14
+    DEFPUSHBUTTON "&Apelare", 1590, 10, 180, 60, 14, WS_GROUP
+    PUSHBUTTON "&Revocare", 1591, 72, 180, 60, 14
```

Comment:
This change introduces an accelerator collision with the line below
PUSHBUTTON "P&roprietăți", 1107, 134, 180, 60, 14
Please check your romanian Windows to find out which of both lines need to be changed.